configure.ac

```diff
@@ -1,6 +1,6 @@
 AC_PREREQ(2.63)
 AC_INIT([libvigor], [m4_esyscmd([./version.sh])], [bugs@niftylogic.com])
-AX_ABI_VERSION([2], [0], [1])
+AX_ABI_VERSION([2], [0], [0])
```

Comment:
it's first minus last for and the last number becomes the middle lib/libvigor.1.1.0, I still don't have a proper mnemonic device to explain it. 

the above change produces
we're going to changing the abi to lib/libvigor.2.0.0 and libvigor.2